src/lib/apis/dashboard/products/delete-product.api.ts

```
@@ -0,0 +1,44 @@
+"use server";
+
+import { JSON_HEADER } from "@lib/constants/shared.constant";
+import { getAuthHeaders } from "@lib/utils/get-auth-headers.util";
```

ESLint

[error] 'getAuthHeaders' is defined but never used.

(@typescript-eslint/no-unused-vars)